Explorer/Assets/Scripts/CrdtEcsBridge/Engine/RuntimeImplementation.cs

```diff
@@ -74,10 +76,35 @@ async UniTask<StreamableLoadingResult<ITypedArray<byte>>> CreateFileRequestAsync
             };
         }
 
+        private static readonly QueryDescription GET_REALM_COMPONENT = new QueryDescription().WithAll<RealmComponent>();
+        private const bool IS_PREVIEW_DEFAULT_VALUE = false;
+
+        public async UniTask<IRuntime.GetRealmResponse> GetRealmAsync(CancellationToken ct)
+        {
+            await UniTask.SwitchToMainThread();
+
+            IRuntime.RealmInfo realmInfo = new IRuntime.RealmInfo();
+
+            world.Query(in GET_REALM_COMPONENT, (ref RealmComponent b) =>
```

Comment:
I think it can't work this way:

- `RealmComponent` exists in the global world, not in every individual scene world
- Also, I believe it's an overkill to query the world for the static information

I propose you to use `IRealmData` instead:
- it exists in a single instance
- it's not a component, you can just pass it to the constructor - it never changes, it gets reconfigured when the realm changes so it's safe to use.

Reply:
Great!